ui/raidboss/data/06-ew/raid/p3n.ts

```diff
@@ -0,0 +1,212 @@
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6698', source: 'Phoinix', capture: false }),
+      response: Responses.getInThenOut(),
+    },
+    {
+      id: 'P3N Experimental Fireplume 2',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6696', source: 'Phoinix', capture: false }),
+      alertText: (_data, _matches, output) => {
+        return output.outOfMiddle!();
+      },
+      outputStrings: {
+        outOfMiddle: {
+          en: 'Out Of Middle',
+          de: 'Raus aus der Mitte',
+          ja: '横へ',
+          cn: '远离中间',
+          ko: '가운데 피하기',
+        },
+      },
+    },
+    {
+      id: 'P3N Scorched Exaltation',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '66B8', source: 'Phoinix', capture: false }),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'P3N Heat of Condemnation',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '66B2', source: 'Phoinix' }),
+      suppressSeconds: 1,
+      response: Responses.tankCleave('alert'),
+    },
+    {
+      id: 'P3N Darkened Fire Aoe',
+      type: 'HeadMarker',
+      netRegex: NetRegexes.headMarker({ id: '010[C-F]' }),
+      condition: Conditions.targetIsYou(),
+      alertText: (_data, _matches, output) => {
+        return output.text!();
+      },
+      outputStrings: {
+        text: {
+          en: 'Stand on Darkened Fire',
+        },
+      },
+    },
+    {
+      id: 'P3N Right Cinderwing',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '66B4', source: 'Phoinix', capture: false }),
+      response: Responses.goLeft(),
+    },
+    {
+      id: 'P3N Left Cinderwing',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '66B5', source: 'Phoinix', capture: false }),
+      response: Responses.goRight(),
+    },
+    {
+      // Could check the log line's x y coordinate to determine from where to where it charges, npc at charge target casts 66AF?
+      id: 'P3N Trail of Condemnation',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '66AD', source: 'Phoinix', capture: false }),
+      alertText: (_data, _matches, output) => {
+        return output.avoidCharge!();
+      },
+      outputStrings: {
+        avoidCharge: {
+          en: 'Avoid Charge',
+          de: 'Charge ausweichen',
+          fr: 'Évitez les charges',
+          ja: '突進避けて',
+          cn: '躲避冲锋',
+          ko: '돌진을 피하세요',
+        },
+      },
+    },
+    {
+      id: 'P3N Sunbird Spawn',
+      type: 'AddedCombatant',
+      netRegex: NetRegexes.addedCombatant({ name: 'Sunbird', capture: false }),
+      suppressSeconds: 1,
+      alertText: (data, _matches, output) => {
+        if (data.role === 'tank')
+          return output.tank!();
+        return output.text!();
+      },
+      outputStrings: {
+        tank: {
+          en: 'Split Adds Don\'t Stack Circles',
```

Comment:
What about:

```suggestion
          en: 'Pull add circles apart',
```

Reply:
Sure, whatever lingo you prefer :) changed in ce1618196bf3bd366946d76a4e8a61c8dde9aeab